fe/src/main/java/org/apache/doris/qe/VariableMgr.java

```diff
@@ -203,6 +209,33 @@ private static void checkUpdate(SetVar setVar, int flag) throws DdlException {
         }
     }
 
+    // Check if the time zone_value is valid
+    private static void checkTimeZoneValid(SetVar setVar) throws DdlException {
+        if (setVar.getValue() != null) {
+            String value = setVar.getValue().getStringValue();
+            try {
+                Pattern p = Pattern.compile("^[+-]{1}\\d{2}\\:\\d{2}$");
+                Matcher m = p.matcher(value);
+                if (!value.contains("/") && !value.equals("CST") && !m.matches()) {
+                    ErrorReport.reportDdlException(ErrorCode.ERR_UNKNOWN_TIME_ZONE, setVar.getValue().getStringValue());
+                }
+                if (m.matches()) {
```

Comment:
Can you add some comments for this function to let people known what it want to do?